e2e/tests/user/registration-ui.spec.ts

```diff
@@ -1,7 +1,5 @@
 import BaseElement from '../../app/aou-elements/base-element';
-import SelectComponent from '../../app/aou-elements/select-component';
-import Textbox from '../../app/aou-elements/textbox';
-import CreateAccountPage, {FIELD_LABEL, INSTITUTION_ROLE_VALUE, INSTITUTION_VALUE} from '../../app/create-account-page';
+import CreateAccountPage from '../../app/create-account-page';
```

Comment:
Can we use module exports or something so we don't have to import from grandparent directories? It's really breakable when you do that, because it means you can't move your test directories around easily.

Reply:
don't know how to do what you suggested. but I will research around for how-to to see what other people are doing.

Reply:
I think it should be possible to specify a root directory, so you can use absolute paths from the root rather than relative from here.